djangocms_snippet/models.py

```diff
@@ -20,13 +20,13 @@ class Snippet(models.Model):
     """
     name = models.CharField(
         verbose_name=_('Name'),
-        unique=True,
         max_length=255,
     )
     snippet_grouper = models.ForeignKey(
         SnippetGrouper,
         on_delete=models.PROTECT,
```

Comment:
This doesn't match the migrations that you ran, did you change the protection bit later i.e. after creating the migrations? 

https://github.com/django-cms/djangocms-snippet/pull/74/files#diff-4ca21242eb21cfa24bd2e3f9e4848d967856c53f97e1cd1838f5dc194efcd14fR21

If you ran make migrations this will create a new migration file. The migrations files will need to be updated from Cascade.